test-poc/framework/pom.xml

```diff
@@ -86,6 +86,10 @@
             <groupId>org.testcontainers</groupId>
             <artifactId>mssqlserver</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
```

Comment:
Do we actually use this? I don't think so as we seem to just start a generic container.

Reply:
We do use it, it's the dependency for `OracleContainer`